### PR TITLE
[SQL Lab] Remove double scrollbar from the result tab

### LINKED ISSUE #7081: [SQL Lab] Remove double scrollbar from the result tab
The result table is currently going offscreen as well.

### PR DESCRIPTION
**Pull request by enricoberti**
closes #7081

### CODE REVIEW

**Comment by mistercrunch** [comment]
@xtinec I know you touched this layout recently, mind reviewing this? I assume this was tested on both FF and Chrome.

Screenshots in layout-related issues/PR are always welcomed!

File: superset/assets/src/SqlLab/components/ResultSet.jsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const defaultProps = {
   database: {},
 };
 
-const SEARCH_HEIGHT = 46;
+const SEARCH_HEIGHT = 186;
```
Comment (xtinec) [inline superset/assets/src/SqlLab/components/ResultSet.jsx]: hmm, I think the hardcode here is the height of the `ResultSetControl` div. It appears to be 46 pixels in both Chrome and FF. Are we sure about 186 here?

File: superset/assets/src/components/FilterableTable/FilterableTable.jsx [hunk 2]
```diff
@@ -189,7 +189,7 @@ export default class FilterableTable extends PureComponent {
           <Table
             ref="Table"
             headerHeight={headerHeight}
-            height={height - 2}
+            height={height - 20}
```
Comment (xtinec) [inline superset/assets/src/components/FilterableTable/FilterableTable.jsx]: why 20 here?

**Comment by xtinec** [comment]
Also, I have a big-ish layout refactoring PR that addresses this and the issues on Firefox. Coming out in the next day or two.
I'll tag you both on it @enricoberti @mistercrunch

File: superset/assets/src/SqlLab/components/ResultSet.jsx (hunk 1 above)
Comment (enricoberti) [reply superset/assets/src/SqlLab/components/ResultSet.jsx]: as it was, the result table was going out of the screen (last row never visible, unless you would scroll the outer panel)

I'm generally against magic numbers in the code though, so we should probably find a proper solution for it (ie calculating the sibling heights after rendering) :)

**Comment by enricoberti** [comment]
@xtinec yes, let's go over this after your PR - i believe this is gonna be superfluous then :) thanks Christine!

**Comment by xtinec** [comment]
@enricoberti yep, I think #7102 supersedes it. 🙌

**Comment by enricoberti** [comment]
@xtinec on the last master I still have the double scrollbar though - it seems that the outer panel it too tall. Do you want me to have a look? Thanks!

20190328_scroll 2019-03-28 12_45_32

**Comment by xtinec** [comment]
@enricoberti Yeah, I'm seeing it as well. Trying to figure out why. I noticed it only happens when there is a horizontal scroll bar, meaning the height of the react virtualized table is off by the scrollbar height. Strangely, I accounted for this in the PR. Not sure what's causing it. 🤔  Any thoughts? 
https://github.com/apache/incubator-superset/blob/71f1bbd2ec59b99d6ba6d9a4a2f9cfceaf922b80/superset/assets/src/components/FilterableTable/FilterableTable.jsx#L186

**Comment by xtinec** [comment]
@enricoberti okay, I just figured it out. 🙂  PR out to you shortly. 🙌

**Comment by enricoberti** [comment]
Thanks @xtinec! Closing this PR as superseded by yours :)

**Pull request closed without merging**